accepted/0000-caasp-integration.md

```diff
@@ -0,0 +1,375 @@
+
+#### Cluster bootstrapping
+
+The cluster must be initiated by the Management Node that has all the tools to bootstrap and manage the cluster. Specific instructions on how to bootstrap the cluster must be provided.
+Questions that need to be answered by the cluster provider:
+- Upon successful bootstrapping, can the worker nodes be managed by SUSE Manager?
+- Do the nodes have a specific agent running on them and/or can we install a `salt-minion` or use `salt-ssh` on those nodes?
+- Can the nodes be managed by SUSE Manager? Can the packages be updated or do they require specific handling by the Management Node?
+
+### Manage a cluster using SUSE Manager
+
+SUSE Manager offers to show the status of the running cluster by either accessing the cluster directly or via the Management Node.
+In this regard, the cluster provider should provide a way to expose the cluster status (e.g. running nodes, the health of each node, monitoring metrics...) to SUSE Manager.
+
+#### Add and removal of nodes to the cluster
+
+SUSE Manager also offers the possibility of adding and removing nodes to the cluster:
+
+- To add a node from scratch, an AutoYaST profile can be reused
+- To remove a node, SUSE Manager must remove the node from the cluster using the Management Node and issuing the proper procedure
+
+SUSE Manager should be instructed to perform additional actions depending on the role of added/removed node (e.g. control plane, worker, load balancer, ...).
+
+#### Cluster upgrade
+
+If every node of the cluster is not directly registered to SUSE Manager, SUSE Manager should offer the possibility to use the Management Node to trigger an update of every node in the cluster, following the standard procedure of upgrading predisposed by the cluster provider.
+
+Also, SUSE Manager must coordinate the Management Node to issue a correct upgrade procedure of the product (e.g. from CaaS Platform version 3 to 4).
+
+#### Application deployment
+
+SUSE Manager can also offer the possibility of deploying an application on top of the cluster deployed (e.g. Kubernetes deployment descriptor to deploy a Kubernetes operator). In this case, the cluster provider must provide the Management Node of the necessary tooling to deploy an application using an API. SUSE Manager makes use of the Management Node to deploy a provided application.
+
+#### Specific usage of SUSE Manager features
+
+One cluster deployment can also leverage SUSE Manager features: SUSE CaaS Platform can trigger a CVE Audit on a container image before deploying it. The cluster provider must outline which features intend to use and which interface is to be used (preferred: XMLRPC API).
+
+## CaaS Platform-specific design
+
+### Deploy a CaaS platform cluster using SUSE Manager
+
+  #### Cluster Deployment
+
+  Under SUSE Manager main menu, a new menu item "CaaS Platform" should be introduced.
```

Comment:
Is this going to be a sub-item under "Clusters"?

Reply:
Yes. Fixed the wording.